scripts/GenerateCredentialSpecResource.ps1

```diff
@@ -37,19 +38,19 @@ Invoke-WebRequest "https://raw.githubusercontent.com/Microsoft/Virtualization-Do
 Import-Module $env:temp\cred.psm1
 
 # generate a unique docker cred spec name
-$dockerCredSpecName = "tmp-k8s-cred-spec" + -join ((48..57) + (97..122) | Get-Random -Count 64 | % {[char]$_})
+$dockerCredSpecName = "tmp-k8s-cred-spec" + -join ((48..57) + (97..122) | Get-Random -Count 64 | ForEach-Object {[char]$_})
```

Comment:
Is this any different? I'm fine with it, just curious :)

Reply:
That was just for clarity.  When I write a PS script that I publish I like to use the cmdlet name instead of the shortcut to make it easier for people (that might not be PS savvy) to read.

Reply:
Rightly so :)